Vancouver-Elsevier-Spain

```diff
@@ -0,0 +1,156 @@
++<?xml version="1.0" encoding="utf-8"?>  
++<style class="in-text" demote-non-dropping-particle="sort-only" version="1.0" page-range-format="minimal" xmlns="http://purl.org/net/xbiblio/csl">  3 +  <info>  
++  <title>Vancouver-Elsevier-Spain</title>  
++    <id>http://www.zotero.org/styles/Vancouver-Elsevier-Spain</id>  
++    <link href="http://www.zotero.org/styles/Vancouver-Elsevier-Spain" rel="self"/>  
```

Comment:
I can help you with that. 
Check out the folder here and the styles in here. They're all little stubs of styles, that really just use another style (independent-parent; that can be your medicina-clinica). https://github.com/citation-style-language/styles/tree/master/dependent
That way we can add all your various styles (Cirugia Española etc...) you outlined in your opening post. ;) easier for the end user as they can just search for their journal and get going.